patches/chromium/fix_font_flooding_in_dev_tools.patch

```diff
@@ -0,0 +1,279 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ryan Manuel <ryanm@cypress.io>
+Date: Fri, 1 Dec 2023 16:40:02 -0600
+Subject: fix font flooding in dev tools
+
+Added in this CL: https://chromium-review.googlesource.com/c/chromium/src/+/5033885
+
+This patch makes the mksnapshot fix available sooner.
```

Comment:
Is this description related to the patch ? Which mksnapshot fix is being referred here ?

Reply:
Oops that was a copy paste from a different patch. Sorry about that. Will fix

Reply:
Updated on this commit: https://github.com/electron/electron/pull/40681/commits/6179518fdc3825a460fac2420c4f17b03bb87033